packages/manager/modules/nasha/src/components/partition/edit-size/edit-size.template.html

```diff
@@ -0,0 +1,21 @@
+<nasha-components-modal-form
+    data-submit=":: $ctrl.submit()"
+    data-close=":: $ctrl.close()"
```

Comment:
Related to this comment:
https://github.com/ovh/manager/pull/6864/files#r868986354